src/utils/arrays.ts

```diff
@@ -0,0 +1,9 @@
+const intersection = (array1: any[], array2: any[]): any[] => array1.filter((item) => array2.indexOf(item) !== -1)
+
+const startsWith = (array: any[], startWithArray: any[]): boolean =>
+  !startWithArray.some((item, index) => array[index] !== item)
```

Comment:
instead of any, use generic. e.g.

const intersection<T> = (array1: T[], array2: T[]): T[] => array1.filter((item) => array2.indexOf(item) !== -1)

Reply:
ok